### ISSUE TITLE
Casting to xsd:long is failing

### ISSUE THREAD

**Issue by GavinMendelGleason** [terminusdb/terminusdb#120]
Casting to xsd:long is failing on the following query.

```javascript
const csv = WOQL.get(
    WOQL.as("first_name","v:first_name")
        .as("last_name", "v:last_name")
        .as("phone_number", "v:phone_number")
        .as("city", "v:city")
        .as("age", "v:age")
).remote("https://raw.githubusercontent.com/graknlabs/examples/master/datasets/phone-calls/people.csv") but then the  Typecast fails WOQL.typecast("v:age", "xsd:long" , "v:age_Cast"),
```

**Comment by rrooij** [comment]
It gives back an empty result, strangely:

```javascript
WOQL.and(WOQL.get(
    WOQL.as("first_name","v:First_name")
        .as("last_name", "v:Last_name")
        .as("phone_number", "v:Phone_number")
        .as("city", "v:city")
        .as("age", "v:age")
).remote("https://raw.githubusercontent.com/graknlabs/examples/master/datasets/phone-calls/people.csv"),
WOQL.typecast("v:age", "xsd:long", "v:age_long"))
```

**Comment by k-tipp** [comment]
I'm experiencing the same with xsd:date and xsd:boolean. Strangely, it works with xsd:dateTime. (all tested with the python client)

**Comment by GavinMendelGleason** [comment]
I'll take a look at this today - it's definitely a bug.

**Comment by GavinMendelGleason** [comment]
Fixed in a2f3ce8
